From beeea4e1a22792ebf8e4857e830bd69d7e30d67c Mon Sep 17 00:00:00 2001
From: Tony LIU <junjie.liu@freescale.com>
Date: Mon, 14 May 2012 15:33:13 +0800
Subject: [PATCH 1468/2463] ENGR00209480-5 mx6sl_usb : gadget: change the
 OTG1_ID pad ctrl

- add pull up ctrl to make the ID to be high by default

Signed-off-by: Tony LIU <junjie.liu@freescale.com>
---
 arch/arm/plat-mxc/include/mach/iomux-mx6sl.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/iomux-mx6sl.h b/arch/arm/plat-mxc/include/mach/iomux-mx6sl.h
index cfa1987..2166779 100755
--- a/arch/arm/plat-mxc/include/mach/iomux-mx6sl.h
+++ b/arch/arm/plat-mxc/include/mach/iomux-mx6sl.h
@@ -960,7 +960,7 @@
 #define MX6SL_PAD_EPDC_PWRCOM__WEIM_WEIM_BCLK                                 \
 		IOMUX_PAD(0x03D0, 0x00E0, 3, 0x0000, 0, NO_PAD_CTRL)
 #define MX6SL_PAD_EPDC_PWRCOM__ANATOP_USBOTG1_ID                              \
-		IOMUX_PAD(0x03D0, 0x00E0, 4, 0x05DC, 0, NO_PAD_CTRL)
+		IOMUX_PAD(0x03D0, 0x00E0, 4, 0x05DC, 0, MX6SL_USDHC_PAD_CTRL)
 #define MX6SL_PAD_EPDC_PWRCOM__GPIO_2_11                                      \
 		IOMUX_PAD(0x03D0, 0x00E0, 5, 0x0000, 0, NO_PAD_CTRL)
 #define MX6SL_PAD_EPDC_PWRCOM__USDHC3_RST                                     \
-- 
1.7.7.4

